README.md

```diff
@@ -65,17 +65,11 @@ cargo run
 ## Demos
 
 ### Krustlet using Akri devices
-Try the [Krustlet using Akri devices demo](./demo-kubelet.md) to test and check out the usage of devices discovered by a WebAssembly Discovery Handler by a Wasm Module running on Krustlet. The result will be the architecture below.
+Try out this [demo](./demo-krustlet.md) of Akri's WebAssembly debug echo Discovery Handler discovering devices on a Krustlet node. Akri's Controller runs as a container on the control plane node, per usual, and deploys a Wasm broker to each discovered device. The result will be the architecture below, where local, Wasm, and containerized components are colored as grey, orange, and purple, respectively.
 
 <img src="./KrustletUsingAkriDevicesDesign.png" alt="Krustlet integration architecture" style="padding-bottom: 10px padding-top: 10px;
 margin-right: auto; display: block; margin-left: auto;"/>
 
-### Akri using devices discovered by a Wasi Discovery Handler
-Try the [Krustlet and Akri Integration demo](./demo-kubelet.md) to test and check out the usage of devices discovered by a WebAssembly Discovery Handler into a regular Akri cluster. The result will be the architecture below.
-
-<img src="./AkriUsingKrustletDevicesDesign.png" alt="Krustlet integration architecture" style="padding-bottom: 10px padding-top: 10px;
-margin-right: auto; display: block; margin-left: auto;"/>
-
```

Comment:
Why are we removing this demo? Think its good to show a scenario where the wasm discovery handler is integrated with container applications.

Reply:
I mention in the PR comment my initial reasoning
> Deletes kubectl demo as it may be confusing to have both. Also, the kubectl demo consists of discovering devices locally on one node and advertising them to another which goes against Akri's paradigm of brokers using devices visible to their node.

What the diagram shows in it is an agent advertising to a node devices that were discovered on another node. That seems off.

Reply:
Fair! Yeah, think its a good idea to cut this part of and anyway think you can get close to the result using the main demo